### DISCUSSION TITLE
How to set a table cell as selectedCell ?

### DISCUSSION THREAD

**Discussion by gethari** [ueberdosis/tiptap#3070] [General Questions]
How to pass `anchorCell / headCell` to setCellSelection() when using a context menu on top of the table ?

I am getting this error. Not sure if I am doing it the right way ?

![image](https://user-images.githubusercontent.com/20477956/183644724-050cc267-88d1-4f7b-b7db-94d45f7a838f.png)

**Comment by Mrdaimi** [comment]
same problem，how did you solve it
